### ISSUE TITLE
Add origin time to quakeml cmt output

### ISSUE THREAD

**Issue by ulbergc** [bakerb845/GFAST#23]

**Comment by clintpells** [comment]
If we could get the whole origin quakeML element like
``` 
      <origin publicID="quakeml:us.anss.org/origin/7000ddw9/mww_trigger">
        <time>
          <value>2021-02-26T11:58:33.2Z</value>
        </time>
        <longitude>
          <value>100.6992</value>
        </longitude>
        <latitude>
          <value>-47.3409</value>
        </latitude>
        <depth>
          <value>15000</value>
        </depth>
      </origin>
```

**Comment by ulbergc** [comment]
Closed with #24
